pkg/scripts/tempesta_installer.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/bash
+#
+# Tempesta FW install script.
+#
+# Copyright (C) 2017 Tempesta Technologies, Inc.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License,
+# or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 59
+# Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+
+declare -r TFW_NAME=`basename $0`
+declare -r LONG_OPTS="help,install,remove,purge"
+declare -r DOWNLOAD_DIR=tfw_downloads
+
+declare -r GITHUB_USER="tempesta-tech"
+declare -r GITHUB_REPO_TEMPESTA="tempesta"
+declare -r GITHUB_REPO_LINUX="linux-4.8.15-tfw"
+
+#TODO: currently Debian 9 is the only supported distribution, other
+# distributions may have other names.
+# Don't install packages with debug symbols.
+declare -a FILES_LINUX=("linux-image-[\d.]+-tempesta-amd64-unsigned"
+                        "linux-kbuild-[\d.]+_[\d.]+"
+                        "linux-headers-[\d.]+-tempesta")
+declare -a FILES_TEMPESTA=("tempesta-fw-dkms")
+
+
+usage()
+{
+	echo -e "\nUsage: ${TFW_NAME} {action}\n"
+	echo -e "Actions:"
+	echo -e "  --install    Install or Update TempestaFW and all dependencies."
+	echo -e "  --remove     Remove TempestaFW, but hold the configuration."
+	echo -e "  --purge      Remove TempestaFW and all configuration files."
+}
+
+# Use github API to get information about latest release.
+#
+# `latest` release shows ONLY releases which are not marked as `prereleases`.
+# Github restricts 60 API requests per hour without autorisation. Enough for
+# our needs.
+tfw_download()
+{
+	repo="${1}"
+	uri="https://api.github.com/repos/$GITHUB_USER/$1/releases/latest"
+
+	links=`curl -s $uri | grep browser_download_url | grep -P $2 | cut -d '"' -f 4`
+
+	if [[ ! "$links" ]]; then
+		# Github api rate limit exceded
+		echo "Github API rate limit exceded: more than 60 requests" \
+		     "during last hour. Try again later."
+
+		exit 2
+	fi
+
+	for file in ${links}
+	do
+		wget -q --show-progress -P $DOWNLOAD_DIR/$repo $file
+	done
+}
+
+tfw_install_packages()
+{
+	repo="${1}"
+	shift
+	files=("${@}")
+
+	echo "Downloading latest packages from github.com/$GITHUB_USER/$repo ..."
+	mkdir -p $DOWNLOAD_DIR/$repo
```

Comment:
Ok, but just mention in the Wiki that there will be `tfw_downloads` directory when the script finishes and that a user should have enough disk space (how much?) to download the packages. (One might think to run it from /tmp, which is sometimes is mounted in ramfs and is limited in size.)